staging/src/k8s.io/kms/go.mod

```diff
@@ -18,15 +16,12 @@ require (
 	golang.org/x/net v0.13.0 // indirect
 	golang.org/x/sys v0.10.0 // indirect
 	golang.org/x/text v0.11.0 // indirect
-	golang.org/x/time v0.3.0 // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20230525234030-28d5490b6b19 // indirect
 	google.golang.org/protobuf v1.31.0 // indirect
 	k8s.io/utils v0.0.0-20230406110748-d93618cff8a2 // indirect
 )
 
 replace (
 	k8s.io/api => ../api
-	k8s.io/apimachinery => ../apimachinery
-	k8s.io/client-go => ../client-go
```

Comment:
is client-go an indirect dependency of `"k8s.io/apimachinery/pkg/util/wait"`?

Reply:
like this: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kms/internal/mock_ratelimit_remote_service.go#L24C1-L24C37

Reply:
You need to make that into a separate go module so it no longer impacts the deps of the kms module.

Reply:
sure.